students/M_Sunday/lesson02/series.py

```diff
@@ -0,0 +1,163 @@
+# lesson 02 - Fibonacci and Lucas Series
+
+
+def fibonacci(n):
+    """
+    This function inputs an integer, n, and returns the nth value
+    of the fibonacci series, where n is the integer entry
+
+    :param n: Integer input value
+
+    :return: nth value in fibonacci series
+    """
+    if n == 1:
+        val = 1
+    else:
+        val = 0
+    val_p = 1
+    val_pp = 0
+
+    for i in range(n-1):
+        if n == 0:
+            val = 0
+        elif n == 1:
+            val = 1
+        else:
+            val = val_pp + val_p
+            val_pp = val_p
+            val_p = val
+
+    return val
```

Comment:
This works but since we know that fibonacci(n) = fibonacci(n-2) + fibonacci(n-1), can you think of a simpler solution that utilizes recursion?